### DISCUSSION TITLE
Translate English speech to Polish subtitles

### DISCUSSION THREAD

**Discussion by alanicja** [openai/whisper#2243] [General]
how to do that or its possible? using whisper .mp4 --language pl --task transcribe its provide wrong translation, but conversely it works from Polish audio to English subtitles

**Comment by glangford** [comment]
whisper will sometimes (but not predictably) translate into languages other than English.

> "Translating into languages other than English wasn't a part of the training objective, so that behavior may not be always reliable."

See also
* https://github.com/openai/whisper/discussions/2225

**Comment by realtechspecs** [comment]
> how to do that or its possible? using whisper .mp4 --language pl --task transcribe its provide wrong translation, but conversely it works from Polish audio to English subtitles

Did you find a solution to your problem?
